docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -67,19 +83,31 @@ This metric is a histogram that shows you the number of seconds taken to send no
 
 > These metrics are not available at present in Grafana Cloud.
 
-## Grafana Mimir
+## Metrics for Grafana Mimir
+
+{{% admonition type="note" %}}
+This metric is available in OSS, on-premise, and Grafana Cloud.
+{{% /admonition %}}
 
-Meta monitoring in Grafana Mimir requires having a Prometheus/Mimir server, or other metrics database, collecting and storing metrics exported by the Mimir ruler.
+To meta monitor in Alertmanager, you need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by the Mimir ruler.
 
-#### cortex_prometheus_rule_evaluation_failures_total
+#### rule_evaluation_failures_total
 
 This metric is a counter that shows you the total number of rule evaluation failures.
 
-## Alertmanager
+## Metrics for Alertmanager
 
-Meta monitoring in Alertmanager also requires having a Prometheus/Mimir server, or other metrics database, collecting and storing metrics exported by Alertmanager. For example, if using Prometheus you should add a `scrape_config` to Prometheus to scrape metrics from your Alertmanager.
+{{% admonition type="note" %}}
+These metrics are available for OSS, on-premise, and some are available in Grafana Cloud.
 
-Here is an example of how this might look:
+Use the data source and Metrics browser in Grafana Cloud called `grafanacloud-usage` that is provisioned for all Grafana Cloud customers to view all available meta monitoring and usage metrics that are available in Grafana Cloud.
+{{% /admonition %}}
+
+To meta monitor in Alertmanager, you need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by Alertmanager.
```

Comment:
```suggestion
To meta monitor the Alertmanager, you need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by Alertmanager.
```

Reply:
note that this is only true for oss or on-prem Enterprise customers. If you're cloud, we provide that database for you. :)